src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx

```diff
@@ -27,6 +28,43 @@ export const RefundTaxiForm = ({ state, send }: RefundTaxiFormProps) => {
 
   return (
     <div>
+      <SectionCard
+        title={t(PageText.Contact.travelGuarantee.refundTaxi.taxiReciet.title)}
+      >
+        <Typo.p textType="body__primary">
+          {t(PageText.Contact.travelGuarantee.refundTaxi.taxiReciet.info)}
+        </Typo.p>
+
+        {/*Should here add a component for using camara in phone directly.*/}
```

Comment:
Is this a TODO that is planned?

Reply:
We haven´t really discussed this feature, but it is included in  `Spec kontaktskjemaer FRAM`, so I choose to say yes.

Reply:
```suggestion
        {/* TODO: Add a component for using camera in phone directly. */}
```